freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -239,7 +239,7 @@ public void setTunnel(Tunnel tunnel) {
         this.tunnel = tunnel;
     }
 
-    public boolean isClusterProxyRegistered() {
+    public Boolean getClusterProxyRegistered() {
         return Boolean.TRUE.equals(clusterProxyRegistered);
     }
```

Comment:
LIke above, this is not a pure getter, so the original signature was more appropriate.

Reply:
see above